sources/Algo/include/Algo.h

```diff
@@ -199,6 +199,15 @@ class GeneratorDefinitionAlgorithm : public NodeAlgorithm {
   void operator()(const NodePtr& node);
```

Comment:
Complete description of algorithm

Reply:
What do you mean ?

Reply:
The doxygen docuentation of the operator() provides a short description of the algorithm. It could be interesting to complete it, at least to say that some input generators can be ignored by this algorithm